src/sagemaker/automl/automl.py

```diff
@@ -78,16 +78,13 @@ def fit(self, inputs=None, wait=True, logs=True, job_name=None):
                 is stored. Or an AutoMLInput object. If a local path is provided, the dataset will
                 be uploaded to an S3 location.
             wait (bool): Whether the call should wait until the job completes (default: True).
-            logs (bool): Whether to show the logs produced by the job.
-                Only meaningful when wait is True (default: True).
+            logs (bool): Whether to show the logs produced by the job. Only meaningful when wait
+                is True (default: True). if `wait` is False, `log` will be set to False as well.
```

Comment:
typo: `logs` missing **s**
 if `wait` is False, `log**s**` will be set to False as well.

Reply:
Thanks! updated!